lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,105 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+
+module.exports = {
+  'desc' : i18n._('Create a credential bundle'),
+  'examples' : [{
+    cmd : 'fhc credential create --bundleName=<bundle-name> --bundleType=<bundle-type> --platform=<platform> --env=<env> --privateKey=<private-key> --certificateAndroid=<certificate-android>',
+    desc : 'Create a credential for Android',
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)